src/statements.md

```diff
@@ -1,5 +1,14 @@
 # Statements
 
+> **<sup>Syntax</sup>**\
+> _Statement_ :\
+> &nbsp;&nbsp; &nbsp;&nbsp; `;`\
+> &nbsp;&nbsp; | [_Item_]\
+> &nbsp;&nbsp; | [_LetStatement_]\
+> &nbsp;&nbsp; | [_ExpressionWithoutBlock_][expression] `;`\
```

Comment:
Ok, moved it down.

It can't be just _Expression_ because `{ { () } let x = 0; }` is valid.  That's why I put the sentence about blocks must have the unit type.  Generally I don't think the grammar should include type checking rules.